src/main/java/org/apache/mesos/hdfs/config/HdfsConfig.java

```diff
@@ -1,22 +1,58 @@
 package org.apache.mesos.hdfs.config;
 
 import com.google.inject.Singleton;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
 import org.apache.hadoop.conf.Configuration;
-import org.apache.hadoop.conf.Configured;
 import org.apache.hadoop.fs.Path;
+import org.apache.mesos.hdfs.util.HDFSConstants;
 
 import java.net.InetAddress;
 import java.net.UnknownHostException;
 import java.util.Properties;
 
+/**
+ * Provides executor configurations for launching processes at the slave leveraging hadoop
+ * configurations
+ */
 @Singleton
-public class SchedulerConf extends Configured {
+public class HdfsConfig {
 
-  public SchedulerConf(Configuration conf) {
+  private Configuration hadoopConfig;
```

Comment:
@elingg  the reason it is named hadoopConfig is that it comes from the org.apache.hadoop.conf package space.
HDFSSchedulerConfig doesn't make sense... It is used in the executors as well.   The meaning of the term configDefinition eludes me, but is the best option of the 2.    This config is used by the entire HDFS framework.  so perhaps MesosHdfsConfig or HdfsFrameworkConfig makes sense for the class name.  I'll go with the latter on the next push and we can discuss from there.